### DISCUSSION TITLE
running STT on an AI accelerator

### DISCUSSION THREAD

**Discussion by poohsen** [coqui-ai/STT#1882] [Q&A]
Hi all,
I'm running STT on a raspberry pi with a custom scorer built for controlling mycroft.AI. Quite pleased with the results except the speed of inference. I'm a bit cloud-averse that's why I thought about using one of those fancy USB sticks that can do very fast inferencing. 

Now, after taking a glance at the STT code and its dependencies, I thought it was based on Tensorflow. But after some more digging and a quick search of the Baidu research paper I'm not so sure anymore. Is it? And if not, why is there a tensorflow submodule in the repo?

I also found a comment in the DeepSpeech issues saying that that the Intel Neural Compute Stick [would be useless](https://github.com/mozilla/DeepSpeech/issues/1346#issuecomment-385349181) for deepspeech due to its reliance on RNNs.
However, when i search for Tensorflow and RNN it seems that this type of NN can be implemented with Tensorflow. 
If somebody could clear this up for me or give me some pointers for futher reading, that would help me a lot.

**Comment by reuben** [comment]
It is based on TensorFlow, so if there's some integration with the Intel Neural Compute Stick you should be able to leverage it.
